source/settingsRing.py

```diff
@@ -2,15 +2,16 @@
 import config
 import synthDriverHandler
 import queueHandler
+import driverHandler
 
-class SynthSetting(baseObject.AutoPropertyObject):
-	"""a numeric synth setting. Has functions to set, get, increase and decrease its value """
-	def __init__(self,synth,setting,min=0,max=100):
-		self.synth=synth
-		self.setting=setting
-		self.min=min
-		self.max=max
-		self.step = setting.normalStep if isinstance(setting,synthDriverHandler.NumericSynthSetting) else 1
+class DriverSetting(baseObject.AutoPropertyObject):
+	"""a numeric driver setting. Has functions to set, get, increase and decrease its value """
+	def __init__(self,driver,setting,min=0,max=100):

```

Comment:
min, max only really make sense for numerical derivatives of this class. I'd prefer it was moved to the specialisation.